.github/workflows/docker.yaml

```diff
@@ -0,0 +1,55 @@
+name: Docker Build
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
**Question**: are we planning to support draft tags such as `0.6.0-dev.1` on this repository?
(It's ok if we don't, just want to be conscious about it.)

Reply:
I don't think we should publish images for those.